File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/maps/LiteModeMapActivity.java
package com.jdroid.android.sample.ui.google.maps;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class LiteModeMapActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return LiteModeFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/maps/LiteModeMapActivity.kt
package com.jdroid.android.sample.ui.google.maps

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class LiteModeMapActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return LiteModeFragment::class.java
    }
}
